osu.Game.Tests/Visual/Menus/TestSceneToolbar.cs

```diff
@@ -91,7 +91,7 @@ public void TestRespectsOverlayActivation(OverlayActivation mode)
 
         public class TestToolbar : Toolbar
         {
-            public new Bindable<OverlayActivation> OverlayActivationMode => base.OverlayActivationMode;
+            public new Bindable<OverlayActivation> OverlayActivationMode => base.OverlayActivationMode as Bindable<OverlayActivation>;
```

Comment:
Would've replaced this with an `IOverlayActivationModeProvider` interface cached to all components tracking overlay activation mode current value, but I guess it's fine if it's only this case.

Reply:
disagree strongly, overengineering

Reply:
On second thoughts, actually yeah, the casting got my eyes on it, but I think it's fine given that this is only a test.